itemspawner/sh_commands.lua

```diff
@@ -0,0 +1,45 @@
+
+local PLUGIN = PLUGIN
+
+ix.command.Add("ItemSpawnerAdd", {
+	description = "@cmdItemSpawnerAdd",
+	arguments = {
+		ix.type.string
+	},
+	OnCheckAccess = function(self, client)
+		return ix.config.Get("spawnerActive", false) and client:IsSuperAdmin()
```

Comment:
You generally shouldn't override `OnCheckAccess` - especially if you're replacing the built-in privilege check with a non-customizable `IsSuperAdmin` call. Check if the config is set in `OnRun` instead.

Additionally, you can group command access together by specifying `COMMAND.privilege`. An example with `/CharBan` and `/CharUnban`: https://github.com/NebulousCloud/helix/blob/master/gamemode/core/sh_commands.lua#L333